packages/amplify-category-api/src/index.ts

```diff
@@ -358,3 +361,14 @@ export const transformCategoryStack = async (context: $TSContext, resource: $TSO
 const canResourceBeTransformed = (
   resourceName: string,
 ): boolean => stateManager.resourceInputsJsonExists(undefined, AmplifyCategories.API, resourceName);
+
+/**
+ * Disable the CDK deprecation warning in production but not in CI/debug mode
+ */
+const disableCDKDeprecationWarning = () => {
+  const isCI = process.env.CI && process.env.CIRCLECI;
```

Comment:
There's a chance that customer is running CLI in headless mode in circle ci.
Is there any way to distinguish between our circle ci and other circle ci ?

Reply:
Emm it will be possible that the customer CI will have these outputs. I notice that the debug model also has a env param `AMPLIFY_ENABLE_DEBUG_OUTPUT`. I think the better way is to remove the CI env check in this line. And also in our CI pipeline, we can add this param in the environment to enable the output.